### PR TITLE
[RS2] Controller visualization updates

### PR DESCRIPTION
**Pull request by keveleigh**
Adds a new prefab with controller visualization built-in.
Fixes an issue where already paired controllers weren't firing a SourceDetected on app launch (resolving some question TODOs).
Fixes an issue where lost controllers weren't properly destroying their visualization GameObjects.

### CODE REVIEW

File: Assets/HoloToolkit/Input/Scripts/ControllerVisualizer.cs [hunk 1]
```diff
@@ -45,6 +46,10 @@ public void OnSourceDetected(SourceStateEventData eventData)
                 {
                     GameObject controller = Instantiate(controllerModel);
                     GameObject parent = new GameObject();
+                    // TODO: Replace this with Controller[Handedness], when possible.
+                    parent.name = "Controller" + Guid.NewGuid().ToString();
+
+                    parent.transform.parent = transform;
```
Comment (fieldsJacksonG) [inline Assets/HoloToolkit/Input/Scripts/ControllerVisualizer.cs]: 😞

File: ProjectSettings/ProjectVersion.txt [hunk 2]
```diff
@@ -1 +1 @@
-m_EditorVersion: 5.6.1f1-MRTP8
+m_EditorVersion: 5.6.1p1-MRTP9
```
Comment (davidkline-ms) [inline ProjectSettings/ProjectVersion.txt]: Do you want to update the projectversion to 2017.2b2?

File: Assets/HoloToolkit/Input/Scripts/ControllerVisualizer.cs (hunk 1 above)
Comment (davidkline-ms) [inline Assets/HoloToolkit/Input/Scripts/ControllerVisualizer.cs]: Handedness is in the 16232 SDK which was flighted a few days back. Unity may need to update for us to see it.

File: ProjectSettings/ProjectVersion.txt (hunk 2 above)
Comment (keveleigh) [inline ProjectSettings/ProjectVersion.txt]: Ahh yes, I've been meaning to try that out in this project.

**Pull request merged**